### ISSUE TITLE
More robust method to specify the 'receiving' node for PTDF/LODF constraint

### ISSUE THREAD

**Issue by manuelma** [Spine-project/SpineOpt.jl#434]
At the moment we are assuming that the second node in `connection__from_node` is the receiving node. This plays well with the way we handle `connection_lossless_bidirectional` but otherwise doesn't feel great. We need a more robust method.

**Comment by DillonJ** [comment]
Agreed - we need a way to define the reference direction somehow (and no, bringing back `direction` doesn't help :-) ) 

Do you have thoughts?

**Comment by manuelma** [comment]
Actually I'm not sure we need it anymore. All we need is a consistent convention as we have (or so it seems to me). Why would the user care?

**Comment by clizbe** [comment]
@manuelma Is this issue still relevant? Otherwise please close it. :)

**Comment by manuelma** [comment]
> @manuelma Is this issue still relevant? Otherwise please close it. :)

Thank you @clizbe for the heads up. Yeah, still relevant I think. Although not too many people uses the PTDF so nobody complains, I think we still should find a better solution.

The current assumption is that the flows goes from the first to the second node in connection__from_node which is kinda obscure.

Cc @DillonJ
